src/middlewares/auth.js

```diff
@@ -11,7 +11,17 @@ async function setCurrentUser(ctx, next) {
   return next();
 }
 
+async function apiSetCurrentUser(ctx, next) {
+  if (ctx.state.authData) {
+    ctx.state.currentUser = await ctx.orm.user.findByPk(
+      ctx.state.authData.userId,
+    );
```

Comment:
Una sugerencia para acceder propiedades del objecto `ctx.state.authData` y también, basado en mi último comentario, debería utilizarse el claim `sub` dentro del payload del JWT:
```suggestion
  const { authData } = ctx.state;
  if (authData) {
    ctx.state.currentUser = await ctx.orm.user.findByPk(authData.sub);
```

Reply:
Cambiado, gracias por la corrección!